From 27e6ba31bb0bea06b4601f26d13c24d9efb9f676 Mon Sep 17 00:00:00 2001
From: Alan Tull <atull@opensource.altera.com>
Date: Wed, 8 Apr 2015 11:46:58 -0500
Subject: [PATCH 127/172] FogBugz #291479: arria10 devkit: slow i2c bus for lcd
 module

Seeing intermittent missing characters on newhaven serial
display so slow the rising/falling time to 6000nSec for
Arria10 devkit.

Signed-off-by: Alan Tull <atull@opensource.altera.com>
[czou:Original patch taken from
https://github.com/altera-opensource/linux-socfpga.git socfpga-4.1]
Signed-off-by: czou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index 8911680..f616b1f 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -206,8 +206,8 @@
 	 * adjust the falling times to decrease the i2c frequency to 50Khz
 	 * because the LCD module does not work at the standard 100Khz
 	 */
-	i2c-sda-falling-time-ns = <5000>;
-	i2c-scl-falling-time-ns = <5000>;
+	i2c-sda-falling-time-ns = <6000>;
+	i2c-scl-falling-time-ns = <6000>;
 
 	lcd: lcd@28 {
 		compatible = "newhaven,nhd-0216k3z-nsw-bbw";
-- 
1.9.1

